extensions/io/org.eclipse.smarthome.io.javasound/src/main/java/org/eclipse/smarthome/io/javasound/internal/JavaSoundAudioSink.java

```diff
@@ -118,4 +136,7 @@ private void runVolumeCommand(Closure closure) {
         }
     }
 
+    private boolean isMacOS() {
+        return System.getProperty("osgi.os").equals("macosx");
```

Comment:
Thanks for the links, I have adapted the code accordingly and it seems to work well.